pages/glossary/hidden-state.md

```diff
@@ -8,15 +8,16 @@ input_aspects:
   - DOM tree
 ---
 
-An HTML element's _hidden state_ is "true" if at least one of the following is true for itself or any of its [ancestors][] in the [flat tree][]:
+An HTML element's _hidden state_ is "true" if either it has a [computed][] CSS property `visibility` whose value is not `visible`; or at least one of the following is true for any of its [inclusive ancestors][] in the [flat tree][]:
 
 - has a `hidden` attribute; or
 - has a [computed][] CSS property `display` of `none`; or
-- has a [computed][] CSS property `visibility` of `hidden`; or
 - has an `aria-hidden` attribute set to `true`
 
 In any other case, the element's _hidden state_ is "false".
 
-[ancestors]: https://dom.spec.whatwg.org/#concept-tree-ancestor 'Definition Ancestor'
+> **Note**: Contrarily to the other conditions, the `visibility` CSS property may be reverted by descendants.
```

Comment:
```suggestion
**Note**: Contrarily to the other conditions, the `visibility` CSS property may be reverted by descendants.
```

Otherwise this will end up in a blockquote.